test/sharness/t0139-swarm-rcmgr.sh

```diff
@@ -227,4 +227,32 @@ test_expect_success 'stop iptb' '
   iptb stop 2
 '
 
+## Test daemon refuse to start if connmgr.highwater < ressources inbound
+
+test_expect_success "node refuse to start if Swarm.ResourceMgr.Limits.System.Conns <= Swarm.ConnMgr.HighWater" '
+  ipfs config --json Swarm.ResourceMgr.Limits.System.Conns 128 &&
+  ipfs config --json Swarm.ConnMgr.HighWater 128 &&
+  ipfs config --json Swarm.ConnMgr.LowWater 64 &&
+  test_expect_code 1 ipfs daemon &&
+  ipfs config --json Swarm.ResourceMgr.Limits.System.Conns 256
```

Comment:
This is because the config is shared with the following tests, so if we want to test each codepath individually we need to reset them above high water.